### ISSUE TITLE
Stuck in GDM / no login prompt

### ISSUE THREAD

**Issue by chillpert** [ublue-os/nvidia#115]
Since I upgraded to a newer version I am stuck in GDM. The grey background is there but there is no login prompt or any other buttons. When switching to another session with Ctrl+Alt+F2, I am able to login using the TUI and `startx` successfully gets me to my desktop. Unfortunately, none of my Flatpaks are launching there though. I tried `systemctl restart gdm` which allowed me to also log in with the original Xorg session but there I can also not launch any Flatpaks. When I restart gdm the TUI greets me with the following error:

```
xf86EnableIOPorts: failed to set IOPL for I/O (Operation not permitted) 
MESA-LOADER: Failed to open simpledrm: /usr/lib64/dri, suffix _dri)
kmsro: driver missing
```

System:
- ublue (Nvidia)
- RTX 2070
- Last known working version `38.20230531.0 (2023-06-01T07:32:58Z)`
- Target version `38.20230607.0 (2023-06-07T11:57:03Z)` 
- no packages layered / default ublue experience

**Comment by didnt1able** [comment]
I just noticed this issue as well. After using `startx` I used `just` to check if cuda was working and it was. Also `nvtop` reports correctly.

Using `systemctl restart gdm` and selecting wayland crashes out and returns to TUI.

**Comment by didnt1able** [comment]
I ran it and rebooted, just get a blackscreen after grub

**Comment by EyeCantCU** [comment]
If you run rpm-ostree upgrade is anything resolved? A user on a custom image told me after rebuilding their image the issue seemed to be gone

**Comment by didnt1able** [comment]
> If you run rpm-ostree upgrade is anything resolved? A user on a custom image told me after rebuilding their image the issue seemed to be gone

Im on :latest so it doesn't pull any changes with upgrade.

**Comment by castrojo** [comment]
Do we need to kick off an image rebuild?

**Comment by EyeCantCU** [comment]
> Do we need to kick off an image rebuild? 

Yes, I think we do. I had a conversation over on Discord related to this earlier and a rebuild of that users image seemed to have addressed the issue

**Comment by yodatak** [comment]
Same for me i got to disable wayland support in gdm to make it work 

in `/etc/gdm/custom.conf `


```ini
# GDM configuration storage

[daemon]
# Uncomment the line below to force the login screen to use Xorg
WaylandEnable=false

[security]

[xdmcp]

[chooser]

[debug]
# Uncomment the line below to turn on debugging
#Enable=true

```

**Comment by yodatak** [comment]
Maybe https://www.nvidia.com/Download/driverResults.aspx/205464/en-us/ will help too  535.54.03
